cmd/notary/tuf.go

```diff
@@ -889,7 +889,13 @@ func tokenAuth(trustServerURL string, baseTransport *http.Transport, gun data.GU
 	if endpoint.Scheme == "" {
 		return nil, fmt.Errorf("Trust server url has to be in the form of http(s)://URL:PORT. Got: %s", trustServerURL)
 	}
-	subPath, err := url.Parse("v2/")
+	var p string
+	if strings.HasSuffix(endpoint.Path, "/") {
```

Comment:
@endophage 
Are you saying user have to call 
```notary -s https://docker.com/notary/```??
Wouldn't it be better if the code can handle both cases, with or without the trailing ```/```